packages/app/src/files-manager/containers/files-manager.container.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import {
+  AnalyticDataTrackerClass,
+  AnalyticDataTrackerFactory,
+} from "App/analytic-data-tracker/services"
+
+const apiUrl = String(process.env.MUDITA_CENTER_SERVER_URL)
+
+let analyticDataTracker: AnalyticDataTrackerClass
+
+export const createFilesManager = () => {
+  if (!analyticDataTracker) {
+    analyticDataTracker = AnalyticDataTrackerFactory.create({
+      apiUrl,
+    })
+  }
+
+  return analyticDataTracker
+}
+
+export const getAnalyticDataTracker = ():
+  | AnalyticDataTrackerClass
+  | undefined => analyticDataTracker
```

Comment:
to remove

Reply:
Removed in [6c450ae](https://github.com/mudita/mudita-center/pull/677/commits/6c450ae5aba2b34e8c2f903ee9bd5141e65f31f0)